app/src/main/res/values/strings.xml

```diff
@@ -177,13 +177,13 @@
     <string name="key_pos_end">Line End (Pos End)</string>
     <string name="key_pos_1_document">Document Start</string>
     <string name="key_pos_end_document">Document End</string>
-    <string name="key_ctrl_a">Select all (Ctrl + A)</string>
+    <string name="key_ctrl_a">Select all (Ctrl+A)</string>
     <string name="key_tab">Tabulator</string>
     <string name="start_on_bottom">Start on bottom</string>
-    <string name="start_on_bottom_summary">After a document gets loaded, position the cursor on the end</string>
-    <string name="filemanager_doesnot_supply_required_data">Markor only allows to edit local, offline available, documents and support file managers which supply the full filepath including filename. Apparently the app didn\'t supply all required information and can\'t be used with Markor :(.</string>
-    <string name="sync_to_local_folder_notice">Notice: If you use a sync app, the files may reside in the known or custom folders, you can open the files by changing Markor\'s document folder to it, or by opening the file via a different file manager.</string>
-    <string name="configure_in_the_apps_settings">Configure in the apps settings</string>
+    <string name="start_on_bottom_summary">Upon loading a document, position cursor at its end</string>
+    <string name="filemanager_doesnot_supply_required_data">Markor can only edit local, offline, documents and file managers supplying full filepath, including filename. The app didn\'t supply all required info and can\'t be used with Markor :(.</string>
+    <string name="sync_to_local_folder_notice">Notice: If you use a sync app, files may reside in custom folders, open them by changing Markor\'s document folder accordingly, or by opening the file in another file manager.</string>
```

Comment:
This is likely problematic, as my understanding of the source string is lacking.

Reply:
then just leave it :)

Reply:
Could you explain the "filemanager_doesnot_supply_required_data" string?